src/corehost/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 cmake_minimum_required (VERSION 2.6)
+include(../settings.cmake)
```

Comment:
Instead of doing this, why dont you include settings.cmake in setup.cmake so that existing usage of setup.cmake inclusion (https://github.com/dotnet/core-setup/search?utf8=%E2%9C%93&q=setup.cmake&type=) remain intact?